example/example.py

```diff
@@ -0,0 +1,8 @@
+from nimiqclient import *
+
+client = NimiqClient()
+
+print(client.consensus())
+
+for account in client.accounts():
+	print(account.address)
```

Comment:
What kind addresses get printed out here?

Reply:
I'll add data structures for using in returned types like for instance [Account](https://github.com/nimiq/core-js/wiki/JSON-RPC-API#account) so that the type of attribute is specified in the docs.

Reply:
`client.accounts()` now returns a mixed list with objects of type [Account](https://github.com/rraallvv/python-client/blob/master/nimiqclient/nimiq_client.py#L48), [VestingAccount](https://github.com/rraallvv/python-client/blob/master/nimiqclient/nimiq_client.py#L67), and [HTLC](https://github.com/rraallvv/python-client/blob/master/nimiqclient/nimiq_client.py#L104).

Given that the IDE can't infer what type is being returned, for instance when traversing the list of accounts, the user still will have to check the docs to have an idea of what attributes each object type has ([see the updated example](https://github.com/rraallvv/python-client/blob/master/example/example.py)).

Reply:
I don't know Python too well, but in other langs, you can create a hierarchy, e.g. the "Account" being on top and "VestingAccount" and "HTLC" being specialisations of "Account". Is that possible? Because if yes, then the method can return a list of "Account" - wdyt? Is that better or not common in Python?

Reply:
[Made](https://github.com/rraallvv/python-client/commit/42ce595100a6529a0feb2dd53327210fd2870e9d) the changes to subclass the VestingContract and HTLC classed from Account